pkg/js/transformer/_testcases.ts

```diff
@@ -56,3 +57,17 @@ export function loadInvalidDslSyntaxTestCases(): InvalidDslSyntaxTestCase[] {
 
   return JSON.parse(jsonData.toString("utf8")) as InvalidDslSyntaxTestCase[];
 }
+
+
+interface DslSyntaxErrorTestCase {
```

Comment:
Use the `InvalidDslSyntaxTestCase` interface. You don't need this

Reply:
Gotcha.

Reply:
Actually, based on the current structure of the test data, it was being used to deal with the current yaml structure, that is undecided on. Primarily I was using this in the tests to count how many error cases were expected.

https://github.com/openfga/language/blob/cc5a21078e403cf44614079078eacda5da27eed4/tests/data/dsl-syntax-validation-cases.yaml#L263-L281

Reply:
Extend the previous interface. In the end all invalid tests need to have these.